packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## V.V.V (Unrealeased)
```

Comment:
It's on you to assign the version, depending on the type of change. It's meant to make the publisher's job easier.

https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs

Since it's a new feature, I'd suggest:

```suggestion
## 7.3.0 (Unreleased)
```